src/parser/OParl/organization.ts

```diff
@@ -0,0 +1,28 @@
+import { isObjectLike } from 'lodash';
+
+import { parseDate } from '../dateParser';
+import { mapToIds } from '../parserHelpers';
+
+// only the necessary validation for parsing is done here.
+// the created database object will be validated after creation before saving
+// by using the .validate method that checks against the provided schema
+export const parseOrganization = (json) => {
+  if (isObjectLike(json)) {
+    const externalId = json.id; // change the id given to us to be the externalId instead
+    delete json.id; // remove the id key from the json to ensure we do not preset it for mongo db
+
+    return {
+      ...json,
+      externalId,
+      created: parseDate(json.created),
+      modified: parseDate(json.modified),
+
+      startDate: parseDate(json.startDate),
+      endDate: parseDate(json.endDate),
+
+      meeting: mapToIds(json.meeting),
+
+      location: json.location?.id,
```

Comment:
which logic are these following having line breaks on specific points?

Reply:
unfortunately i could not immediately think of a very specific well defined logic.

i just sprinkled line breaks in there, where it felt appropriate, e.g. when we have a different type (date vs string) or something of that sort.

it made it way more readable for me even though the line breaks might not be optimal, when comparing it to no line breaks at all.

if you can think of an easy rule here, that yields at least similar results in terms of readability, i would gladly apply it

Reply:
ok, i have expected something like this. me reading this code could just guess types according to names. so it needs some more effort to sort in some stuff with checking all types to put it right.
i cannot think of an applicable rule here. so let's go with the "more readable for me".